### PR TITLE
[CPDLP-2295] Add authorise for payment button to ECF/NPQ on financial statements

### PR DESCRIPTION
**Pull request by leandroalemao**
### Context

- Ticket: [CPDLP-2295](https://dfedigital.atlassian.net/browse/CPDLP-2295)

### Changes proposed in this pull request

Add authorise for payment button to ECF/NPQ on financial statements.

[CPDLP-2295]: https://dfedigital.atlassian.net/browse/CPDLP-2295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by leandroalemao** [comment]
Hi @cwrw I believe I've addressed all your comments.. tks for your review as always.. re:  calculation rounding message.. I had spoken to Mark to do all together as it was only a small copy to add.

File: app/helpers/finance_helper.rb [hunk 1]
```diff
@@ -50,4 +50,12 @@ def latest_induction_record_for_provider?(induction_record, participant_profile)
 
     induction_record == participant_profile.latest_induction_record_for(cpd_lead_provider:)
   end
+
+  def authorise_for_payment_button_visible?(statement)
```
Comment (cwrw) [inline app/helpers/finance_helper.rb:54]: ah I think one last thing to add is hiding the button if `mark_as_paid_at` is set but still payable (after a job is triggered but pending)

**Review by cwrw — COMMENTED** [review]
thank you @leandroalemao for addressing all comments! Looks fantastic, only one last thing I spotted while testing, but after that should be all good to go!

File: app/helpers/finance_helper.rb (hunk 1 above)
Comment (leandroalemao) [reply app/helpers/finance_helper.rb:54]: All done @cwrw I hope it's all good.. tks for your review..

**Review by cwrw — APPROVED** [review]
thanks @leandroalemao looks great!

**Pull request merged**